### ISSUE TITLE
Crashed: NSOperationQueue 0x283f32740 (QOS: UNSPECIFIED)

### ISSUE THREAD

**Issue by BobbyRohweder** [onevcat/Kingfisher#1277]
### Check List

- [x] I have read the [wiki page](https://github.com/onevcat/Kingfisher/wiki) and [cheat sheet](https://github.com/onevcat/Kingfisher/wiki/Cheat-Sheet), but there is no information I need.
- [x] I have searched in [existing issues](https://github.com/onevcat/Kingfisher/issues?utf8=✓&q=is%3Aissue), but did not find a same one.
- [x] I want to report a problem instead of asking a question. It'd better to use [kingfisher tag in Stack Overflow](http://stackoverflow.com/questions/tagged/kingfisher) to ask a question.

#### What

Received a crash in Crashlytics.

#### Reproduce

Unsure how it is caused currently.

#### Other Comment

Using Kingfisher version 5.7.1

```
Crashed: NSOperationQueue 0x283f32740 (QOS: UNSPECIFIED)
0  libswiftCore.dylib             0x1ef713ba0 specialized _assertionFailure(_:_:file:line:flags:) + 360
1  libswiftCore.dylib             0x1ef5372bc _assertionFailure(_:_:file:line:flags:) + 32
2  libswiftFoundation.dylib       0x1ef991bfc specialized Data.InlineSlice.append(contentsOf:) + 1720
3  libswiftFoundation.dylib       0x1ef9924d4 specialized Data._Representation.append(contentsOf:) + 320
4  Kingfisher                     0x104d62038 SessionDelegate.urlSession(_:dataTask:didReceive:) + 117 (SessionDataTask.swift:117)
5  Kingfisher                     0x104d623cc @objc SessionDelegate.urlSession(_:dataTask:didReceive:) (<compiler-generated>)
6  CFNetwork                      0x1c1a6e944 __67-[NSURLSession delegate_dataTask:didReceiveData:completionHandler:]_block_invoke_2 + 40
7  Foundation                     0x1c1d4fec8 __NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__ + 16
8  Foundation                     0x1c1c5c2e0 -[NSBlockOperation main] + 72
9  Foundation                     0x1c1c5b7c8 -[__NSOperationInternal _start:] + 740
10 Foundation                     0x1c1d51c4c __NSOQSchedule_f + 272
11 libdispatch.dylib              0x1c0d28a38 _dispatch_call_block_and_release + 24
12 libdispatch.dylib              0x1c0d297d4 _dispatch_client_callout + 16
13 libdispatch.dylib              0x1c0d00c34 _dispatch_continuation_pop$VARIANT$armv81 + 404
14 libdispatch.dylib              0x1c0d00314 _dispatch_async_redirect_invoke + 592
15 libdispatch.dylib              0x1c0d0c9d4 _dispatch_root_queue_drain + 340
16 libdispatch.dylib              0x1c0d0d248 _dispatch_worker_thread2 + 116
17 libsystem_pthread.dylib        0x1c0f0a1b4 _pthread_wqthread + 464
18 libsystem_pthread.dylib        0x1c0f0ccd4 start_wqthread + 4
```

**Comment by onevcat** [comment]
We've fixed a queue issue in the latest 5.8.0. Can you try to upgrade to the new version to see whether it is resolved? Thanks!

**Comment by BobbyRohweder** [comment]
Thanks, I'll let you know if we see another issue
